Utils/request_contributor_review.py

```diff
@@ -187,20 +221,138 @@ def check_reviewers(reviewers: set, pr_author: str, version: str, modified_files
         print(f'{pack} pack no reviewers were found.')
 
 
+def notify_contributors_by_email(dev_reviewers_emails: Union[str, list], support_reviewers_emails: Union[str, list],
+                                 email_refresh_token: str, pack: str, pr_number: str):
+    """ Checks arguments and finds emails of the reviewers. send mail to developers if there are dev-mails,
+    else send mail to pack support.
+
+    Args:
+        dev_reviewers_emails(Union[str, list]): mails of developers, if there are in pack_metadata
+        support_reviewers_emails(Union[str, list]): mails of support, if there are in pack_metadata
+        email_refresh_token(str): refresh token to send mails using gmail API
+        pack(str): pack that was modified
+        pr_number(str): github pr number
+
+    """
+    dev_emails = ','.join(dev_reviewers_emails) if isinstance(dev_reviewers_emails, list) else dev_reviewers_emails
+    support_emails = ','.join(support_reviewers_emails) if isinstance(support_reviewers_emails, list) \
+        else support_reviewers_emails
+
+    # send mail to developers if there are dev-mails, else send mail to pack support
+    reviewers_emails = dev_emails if dev_emails else support_emails
+    if reviewers_emails:
+        send_email_to_reviewers(
+            reviewers_emails=reviewers_emails,
+            refresh_token=email_refresh_token,
+            pack_name=pack,
+            pr_number=pr_number,
+        )
+
+
+def send_email_to_reviewers(reviewers_emails: str, refresh_token: str, pack_name: str,
+                            pr_number: str):
+    """ Compose mail and send it to the reviewers_emails, to review the changes in their pack
+
+    Args:
+        reviewers_emails(str): reviewers of the pack to send mail to them
+        refresh_token(str): refresh token to send mails using gmail API
+        pack_name(str): pack that was modified
+        pr_number(str): github pr number
+
+    """
+    access_token = get_access_token(refresh_token)
+    credentials = AccessTokenCredentials(access_token, 'Demisto Github send mails to contributors')
+    service = build('gmail', 'v1', credentials=credentials)
+
+    email_content = f"Changes were made to the {pack_name} content pack that you contributed. \n"
+    email_content += f"Please review the changes in the following files in the contribution "
+    email_content += f"[Pull Request](https://github.com/demisto/content/pull/{pr_number}/files)."
```

Comment:
Let's the same text used in the GitHub comment with an addition of a small greeting header like: `Hi {user}` https://github.com/demisto/content/blob/50b3b08dab0646319f76f00e27295da8369bb38a/Utils/request_contributor_review.py#L99-L104

BTW this is the way to build a multiline string in python ☝️